app/views/posts/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<h5> Create a new Post:</h5>
+<%= form_with model: @post, url: user_posts_path do |form| %>
+   <div class="form-group">
+    <%= form.label :title , "Title" %>
+    <%= form.text_field :title %>
+   </div>
+   <div class="form-group">
+    <%= form.label :text , "Text" %>
+    <%= form.text_area :text , size: '40*40'%>
+   </div> 
+    <%= form.submit "Post" %>
+<% end %>
```

Comment:
[OPTIONAL] Since there is no link or button to create comments and posts, the user have to manually type in the link in the browser before you can access the form. The same applies to liking a post. I strongly believe your app will look more perfect and user-friendly if you could add a link or button that will link the user to the form to create a post or comment (as well as liking a post). pray